packages/twenty-front/src/modules/workflow/search-variables/utils/getVariableTag.ts

```diff
@@ -0,0 +1,50 @@
+import { Node } from '@tiptap/core';
+import { mergeAttributes, RawCommands } from '@tiptap/react';
+
+export const getVariableTag = () =>
+  Node.create({
+    name: 'variableTag',
+    group: 'inline',
+    inline: true,
+    atom: true,
+
+    addAttributes: () => ({
+      variable: {
+        default: null,
+      },
```

Comment:
I think it's considered best practice to prefix our custom HTML attributes with `data-`. It establishes a clear distinction between native and custom attributes.

I read how to achieve it here: https://tiptap.dev/docs/editor/extensions/custom-extensions/extend-existing#attributes

```suggestion
      variable: {
        default: null,
        parseHTML: (element) => element.getAttribute('data-variable'),
        renderHTML: (attributes) => {
          return {
            'data-variable': attributes.variable,
          };
        },
      },
```

Reply:
thanks for looking into it! Updated